### DISCUSSION TITLE
Can't compile RustDesk 1.3.1 because of hwcodec 0.7.0

### DISCUSSION THREAD

**Discussion by Pollote-Dax** [rustdesk/rustdesk#9607] [Q&A] [answered]
Hi everyone,
I can compile version 1.2.7 witch pointed de hwcodec dependency to : https://github.com/21pages/hwcodec and version downloaded was 0.6.0.
Now in version 1.3.1 it pointed to https://github.com/rustdesk-org/hwcodec and version downloaded is 0.7.0.
Building it all with the --hwcodec parameter shows warning: hwcodec@0.7.0: ToolExecError .. (status code exit code: 2) and error: failed to run custom build command for `hwcodec v0.7.0 (https://github.com/rustdesk-org/hwcodec#f74410ed).
Then 4 missing .h files and finally some vs2022 cl.exe errors.
Any advice ?
Thanks in advanced.

**Comment by armgadge** [comment]
Have you installed ffmpeg package of vcpkg?
I have never built on windows, but I have an idea with that error.

**Reply by Pollote-Dax** [reply, marked as answer]
Yes I did, but your question lead me to the solution.
I installed the dynamic version instead of static.
./vcpkg install ffmpeg:x64-windows-static solves de issue.
Thanks for your inspiration !
